cmd/root.go

```diff
@@ -30,15 +30,19 @@ func NewMethodK8s(version string) *MethodK8s {
 	methodK8s := MethodK8s{
 		version: version,
 		RootFlags: config.RootFlags{
-			Quiet:          false,
-			Verbose:        false,
-			Context:        "",
-			Path:           "",
-			URL:            "",
-			ServiceAccount: false,
-			Token:          "",
-			CACert:         "",
-			APIServerURL:   "",
+			Quiet:   false,
+			Verbose: false,
+			ServiceAccountConfig: config.ServiceAccountConfig{
+				ServiceAccount: false,
+				Token:          "",
+				CACert:         "",
+				APIServerURL:   "",
```

Comment:
Why do we have `APIServerURL` here and also `URL` in the other struct? Can't we consolidate these?